subsys/bluetooth/audio/bap_broadcast_assistant.c

```diff
@@ -57,14 +59,19 @@ struct bap_broadcast_assistant_instance {
 	struct bt_gatt_read_params read_params;
 	struct bt_gatt_write_params write_params;
 	struct bt_gatt_discover_params disc_params;
+
+	struct k_work_delayable bap_read_work;
+	struct bt_conn *conn;
+	uint16_t long_read_handle;
 };
 
 static struct bt_bap_broadcast_assistant_cb *broadcast_assistant_cbs;
 
 static struct bap_broadcast_assistant_instance broadcast_assistant;
 static struct bt_uuid_16 uuid = BT_UUID_INIT_16(0);
 
-NET_BUF_SIMPLE_DEFINE_STATIC(cp_buf, CONFIG_BT_L2CAP_TX_MTU);
+#define CP_BUF_SIZE BT_ATT_MAX_ATTRIBUTE_LEN
+NET_BUF_SIMPLE_DEFINE_STATIC(cp_buf, CP_BUF_SIZE);
```

Comment:
```suggestion
NET_BUF_SIMPLE_DEFINE_STATIC(att_buf, BT_ATT_MAX_ATTRIBUTE_LEN);
```

Suggest to rename it as it is no longer just used for the control points (CP) but all GATT/ATT operations